tests/kernel/install_ltp.pm

```diff
@@ -280,7 +280,7 @@ sub setup_network {
     assert_script_run('sed -i \'s/^\(hosts:\s+files\s\+dns$\)/\1 myhostname/\' /etc/nsswitch.conf');
 
     foreach my $service (qw(auditd dnsmasq nfs-server rpcbind vsftpd)) {
-        if (is_sle('12+') || is_opensuse || is_jeos) {
+        if ((!is_jeos) && is_sle('12+') || is_opensuse) {
```

Comment:
nit: could be without brackets around `!is_jeos`, i.e. `if (!is_jeos && is_sle('12+') || is_opensuse) {`

Reply:
Updated, thanks!